enterprise/admin/__init__.py

```diff
@@ -0,0 +1,4 @@
+"""
+Module imports admin panels needed for Enterprise app.
+"""
+from enterprise.admin.admin import EnterpriseCustomerAdmin, EnterpriseCustomerUserAdmin
```

Comment:
@haikuginger warning: you're going to have a significant conflict here, as I've kept most of `admin.py` in `admin/__init__.py`. Not sure how best to address it, but I'm ok with both variants (i.e. keeping admins in __init__.py or moving them into a dedicated module and importing from __init__.py)